lib/apiservers/engine/backends/volume.go

```diff
@@ -36,11 +37,20 @@ import (
 
 // NOTE: FIXME: These might be moved to a utility package once there are multiple personalities
 const (
-	OptsVolumeStoreKey     string = "VolumeStore"
-	OptsCapacityKey        string = "Capacity"
+	OptsVolumeStoreKey     string = "volumestore"
+	OptsCapacityKey        string = "capacity"
 	dockerMetadataModelKey string = "DockerMetaData"
 )
 
+// define a set (whitelist) of valid driver opts keys for command line argument validation
+var validDriverOptsKeys = map[string]bool{
```

Comment:
you might consider just using struct{} instead of true. This is a common go idiom for making a set :)